index.js

```diff
@@ -72,9 +72,34 @@ var browserslist = function (selections, opts) {
         selections = selections.split(/,\s*/);
     }
 
+    var statsFile, statsExist;
+
     if ( opts.stats || process.env.BROWSERSLIST_STATS ) {
+        statsExist = true;
+    } else {
+        (function searchStatsFile(startPath) {
```

Comment:
@ai but as I understood from task, it should to search file in any folder of project folder. Or same `while` loop as in config search will be enough?